### PR TITLE
Adds disable constraints and leak hunter tests for MSSQL Server 

### LINKED ISSUE #125: Adds leak hunter and disable constraints for MSSQL server

### PR DESCRIPTION
**Pull request by rmpestano**
#fixes #125

### CODE REVIEW

**Comment by rmpestano** [comment]
Hi @xpinjection,

can you have a look? mainly [this](https://github.com/database-rider/database-rider/pull/126/files#diff-91367f3f37f42c0105bc9f9dd65dad9bR329) and the 'uppercase' `[here](https://github.com/database-rider/database-rider/pull/126/files#diff-91367f3f37f42c0105bc9f9dd65dad9bR643)` because on the test containers image of MSSQL the sys schema is in lowecase .

Thank you in advance.

**Comment by xpinjection** [comment]
Thank you for pointing out, definitely must take into account case sensitivity. I will submit fix and some additional tests. Sorry, I’m new to the codebase, so will do my best to take such nuances into account next time.

**Comment by rmpestano** [comment]
Hi @xpinjection, no need for sorry, you already did a fantastic work here! 

I just need your help on this PR (note that the commits are on my user repo), you can commit together if you want or just review it.

Again, thank you so far!

**Comment by rmpestano** [comment]
If you're ok with the changes I'll merge and release a new dbrider version

File: rider-core/src/main/java/com/github/database/rider/core/dataset/DataSetExecutorImpl.java [hunk 1]
```diff
@@ -389,12 +392,15 @@ public void enableConstraints() throws SQLException {
                     case MSSQL:
                         tables = getTableNames(getRiderDataSource().getConnection());
                         for (String tableName : tables) {
+                            if(tableName.toLowerCase().endsWith("user")) {
```
[the remark this answers is not in the document]
Comment (rmpestano) [reply rider-core/src/main/java/com/github/database/rider/core/dataset/DataSetExecutorImpl.java]: Got it!

**Pull request merged**